apps/openchallenges/challenge-service/src/main/resources/application.yml

```diff
@@ -18,3 +18,4 @@ spring:
       db_incentives_csv_path: ${DB_INCENTIVES_CSV_PATH:src/main/resources/db/incentives.csv}
       db_submission_types_csv_path: ${DB_SUBMISSION_TYPES_CSV_PATH:src/main/resources/db/submission_types.csv}
       db_categories_csv_path: ${DB_CATEGORIES_CSV_PATH:src/main/resources/db/categories.csv}
+      db_edam_concept_csv_path: ${DB_EDAM_CONCEPT_CSV_PATH:src/main/resources/db/edam_concept.csv}
```

Comment:
For my own clarify, the automated CSV dumping script should no longer dump [this sheet](https://docs.google.com/spreadsheets/d/1XRSUVqbZk_k9_rV1vpgPVMQ0fVV0mavwY8Aqyy0C2HQ/edit#gid=1849016276), correct?

Reply:
Correct